examples/talks/Overview.ipynb

```diff
@@ -2,11 +2,7 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "metadata": {
-    "slideshow": {
-     "slide_type": "skip"
-    }
-   },
```

Comment:
This metadata is important; can you please restore all of it for this notebook? This notebook works as a slideshow (or at least it used to), which it can't do without that metadata. We should update the jq stripping command to ignore slideshow metadata.

Reply:
Oh oh I had missed that, thanks! I reverted it in https://github.com/holoviz/holoviz/pull/301/commits/fdec9e1a0440985432b598fde9e711cbe97e0183